pkg/network/payload/ping.go

```diff
@@ -0,0 +1,40 @@
+package payload
+
+import (
+	"time"
+
+	"github.com/CityOfZion/neo-go/pkg/io"
+)
+
+// Ping payload for ping/pong payloads.
+type Ping struct {
+	// Index of the last block.
+	LastBlockIndex uint32
+	// Timestamp.
+	Timestamp      uint32
+	// Nonce of the server.
+	Nonce          uint32
+}
+
+// NewPing created new Ping payload.
+func NewPing(blockIndex uint32, nonce uint32) *Ping {
+	return &Ping{
+		LastBlockIndex: blockIndex,
+		Timestamp:      uint32(time.Now().UTC().Unix()),
+		Nonce:          nonce,
+	}
+}
+
+// DecodeBinary implements Serializable interface.
+func (p *Ping) DecodeBinary(br *io.BinReader) {
+	br.ReadLE(&p.LastBlockIndex)
+	br.ReadLE(&p.Timestamp)
+	br.ReadLE(&p.Nonce)
+}
+
+// EncodeBinary implements Serializable interface.
+func (p *Ping) EncodeBinary(bw *io.BinWriter) {
+	bw.WriteLE(&p.LastBlockIndex)
```

Comment:
`bw.WriteLE(p.LastBlockIndex)`

Reply:
Just add a test for encode/decode, it will fail now.

Reply:
sure, will add tests